codetransformer/transformers/exc_patterns.py

```diff
@@ -0,0 +1,51 @@
+from sys import exc_info
+
+from dis import dis
+
+from codetransformer.core import (
+    CodeTransformer,
+    Instruction,
+    ops,
+)
+
+
+def match(match_expr, exc_info):
+    """
+    Called to determine whether or not an except block should be matched.
+
+    True -> enter except block
+    False -> don't enter except block
+    """
+    return (
+        type(match_expr) == exc_info[0] and
+        match_expr.args == exc_info[1].args
+    )
+
+
+class PatternMatchedExceptions(CodeTransformer):
```

Comment:
I'm not sure why, but I've been feeling the lowercase type names for these, maybe because they are more like features and not classes.